src/scancode/cli.py

```diff
@@ -382,6 +384,7 @@ def scan_event(item):
                     return ''
                 if item:
                     _scan_success, _scanned_path = item
+                    _scanned_path = shorten_filename(_scanned_path)
```

Comment:
this should only be done if we are not verbose. Note also that you are never passing a filename but a path. So your assumptions are not right in your function.